rpm-ostree-bisect

```diff
@@ -129,6 +129,18 @@ def get_deployed_base_commit(deployment, repo):
         commitid = OSTree.commit_get_parent(commit)
     return commitid
 
+"""
+    On newer rpm-ostree systems /sysroot has been mounted read-only
+    which means we can't pull commit metadata into the repo. Let's
+    remount it read-write if that's the case.
```

Comment:
Hmm, ostree should automatically remount rw. Ohhh right, you're doing this via the API. Good for you. :)

So I think what we want to do here is use `MountFlags=slave` in the service unit and then in `main()` before `sysroot.load()`, call out to https://lazka.github.io/pgi-docs/OSTree-1.0/classes/Sysroot.html#OSTree.Sysroot.set_mount_namespace_in_use.

Reply:
The only problem with this is that the very first invocation of rpm-ostree-bisect runs outside of a systemd unit so we'd still have to do one remount rw. Any elegant ideas around that?

Reply:
Easiest is to just call out to `ostree pull` instead of using the API. (Then you don't need to do any of the things in my initial comment either).

Reply:
I'm sure you can find a way to use `unshare()` from Python (ahh yup looks like there's [a package](https://pypi.org/project/unshare/) for that), though honestly I don't think it's worth it.

Reply:
ok fixed in the latest push